src/CoreWCF.Http/tests/HttpSysTests.cs

```diff
@@ -13,6 +14,7 @@
 
 namespace CoreWCF.Http.Tests
 {
+    [Collection(HttpSysTestCollectionDefinition.HttpSysTestCollection)]
```

Comment:
What does the Collection here do to enable concurrent execution?

Reply:
Default Xunit behavior is to run tests from a single TestCollection sequentially. I'm using this behavior to prevent HttpSys tests to run concurrently because when passing port 0 it asks for a port in an admin range. As only a few UT use HttpSys I thought it's acceptable to have this collection run sequentially but the other TestCollection run concurrently.